main.tex

```diff
@@ -447,7 +446,7 @@ \subsubsection{Logarithmic units and magnitudes}
 \subsubsection{Defining functions that require quantities}
         When writing code or
         functions that expect \texttt{Quantity} objects, we often want to
-        enforce that the input units have the correct type.
+        enforce that the input units have the correct physical type.
```

Comment:
Note: "type" is too cryptic. I *think* physical type is accurate here?

Reply:
Yep!